lib/AST/Decl.cpp

```diff
@@ -5824,20 +5851,19 @@ void ParamDecl::setDefaultArgumentInitContext(Initializer *initContext) {
   DefaultValueAndFlags.getPointer()->InitContext = initContext;
 }
 
+/// Return nullptr if there is no property wrapper
 Expr *swift::findOriginalPropertyWrapperInitialValue(VarDecl *var,
                                                      Expr *init) {
   auto *PBD = var->getParentPatternBinding();
   if (!PBD)
     return nullptr;
 
-  // If there is no '=' on the pattern, there was no initial value.
-  if (PBD->getPatternList()[0].getEqualLoc().isInvalid()
```

Comment:
I expected that we would have kept the `getEqualLoc().isInvalid()` check. I think we still need it, because if nothing else it lets us bail out much earlier (before evaluating other requests).

Reply:
Yeah, it's back!